samples/memory/agentcore_valkey_checkpointer.ipynb

```diff
@@ -0,0 +1,938 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 🚀 AgentCore Valkey Checkpointer Usage Example\n",
+    "\n",
+    "This notebook demonstrates how to use the **AgentCoreValkeySaver** for high-performance checkpoint storage that combines AgentCore session management concepts with Valkey's fast storage capabilities.\n",
+    "\n",
+    "## 📋 Prerequisites\n",
+    "\n",
+    "Before running this notebook, ensure you have:\n",
+    "\n",
+    "1. **Valkey server** running on `localhost:6379`\n",
+    "2. **AWS credentials** configured for Bedrock access\n",
+    "3. **Required packages** installed:\n",
+    "   ```bash\n",
+    "   pip install valkey langchain-aws langchain langgraph\n",
+    "   ```\n",
+    "\n",
+    "## 🎯 Key Features\n",
+    "\n",
+    "- 🚀 **High-performance storage** with Valkey\n",
+    "- 🎯 **AgentCore-compatible** session management\n",
+    "- 🔄 **TTL support** for automatic cleanup\n",
+    "- 🏊 **Connection pooling** for scalability\n",
+    "- 🔍 **Metadata filtering** and querying\n",
+    "- 🧹 **Easy cleanup** and management"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 📦 Import Dependencies\n",
+    "\n",
+    "First, let's import all the necessary libraries:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "✅ All dependencies imported successfully!\n"
+     ]
+    }
+   ],
+   "source": [
+    "import time\n",
+    "from typing import Dict, Any\n",
+    "\n",
+    "from langchain.chat_models import init_chat_model\n",
+    "from langchain.tools import tool\n",
+    "from langgraph.prebuilt import create_react_agent\n",
```

Comment:
`create_react_agent` is deprecated in LangGraph V1, we should be using `create_agent` instead (see docs here: https://docs.langchain.com/oss/python/releases/langchain-v1#create-agent)
```suggestion
    "from langchain.agents import create_agent\n",
```

@seaofawareness please replace all usages in this notebook, and regenerate the outputs to remove the deprecation warnings.